ui/admin/app/components/users/user/actions/index.hbs

```diff
@@ -23,4 +23,25 @@
       {{t 'resources.user.actions.delete'}}
```

Comment:
hmm were the changes in this file meant to be part of the PR ?

Reply:
Ahh thanks for the catch Lis🙏🏾 no they weren't; the changes were for another Jira story (ICU-14343), I think I didn't set the branch however. I'll go ahead and revert them to be clear on each